stories/header.stories.tsx

```diff
@@ -1,14 +1,27 @@
-import * as React from 'react';
-import { storiesOf } from '@storybook/react';
+import * as React from "react";
+import { storiesOf } from "@storybook/react";
 
-import Header from '../src/components/Header';
+import Header from "../src/components/Header";
 
-const header = storiesOf('Components/Header', module)
-  .add('with title', () =>
+const header = storiesOf("Components/Header", module)
+  .add("with title", () =>
     <Header text="SimplyE" />
   )
-  .add('with logout link', () =>
+  .add("with logout link", () =>
     <Header text="SimplyE" loggedIn={true} />
+  )
+  .add("with navigation", () =>
+    <Header
+      text="SimplyE"
+      loggedIn={true}
+      nav={
+        <ul>
+          <li><a href="/about">About</a></li>
+          <li><a href="/account">account</a></li>
+          <li><a href="/misc">misc</a></li>
+        </ul>
+      }
+    />
   );
 
```

Comment:
Was there a reason we didn't include an example of a header with a logo?

Reply:
I think I had issues with Storybook/webpack extracting the image url and rendering it. We could possibly use another generic logo somehow.

Reply:
Yeah, the `require(...)` doesn't work in this compilation and it can't find the logo, even when trying different directories. It's a bigger issue for reusability we can tackle later.